src/apply_conf.rs

```diff
@@ -302,6 +309,7 @@ mod tests {
                     logical_name: "eth0".to_string(),
                     mac_address: Option::from("00:11:22:33:44:55".to_string()),
                     interface_type: "ethernet".to_string(),
+                    connection_ids: None,
```

Comment:
Can we use valid examples? In real usages this wouldn't be empty. Applies to the other cases down below too.

Reply:
Sure, I missed this apparently